pkg/cli/server/server.go

```diff
@@ -113,6 +122,15 @@ func run(app *cli.Context, cfg *cmds.Server) error {
 	serverConfig.ControlConfig.StorageCAFile = cfg.StorageCAFile
 	serverConfig.ControlConfig.StorageCertFile = cfg.StorageCertFile
 	serverConfig.ControlConfig.StorageKeyFile = cfg.StorageKeyFile
+	serverConfig.ControlConfig.AdvertiseIP = cfg.AdvertiseIP
```

Comment:
is there any use for AdvertiseIP than adding it to the knownIP, I think we can just get rid of ControlConfig.AdvertiseIP and just default AdvertiseIP to 127.0.0.1

Reply:
AdvertiseIP should not be 127.0.0.1 anymore, it needs to be unique for multiple kubernetes service endpoints & HA to work